source/openapi-admin-v3.yaml

```diff
@@ -3827,72 +3858,152 @@ components:
+      allOf:
+        - $ref: "#/components/schemas/App"
+        - type: object
+          properties:
+            template_id:
+              type: string
+              description: |-
+                The id of an [App Services template
+                app](https://www.mongodb.com/docs/realm/manage-apps/create/template-apps#std-label-template-apps)
+                to use as a base for the new app.
+            data_source:
+              "$ref": "#/components/schemas/DataSource"
+    AppInstance:
+      description: A specific, existing application.
+      allOf: 
+        - $ref: "#/components/schemas/App"
+        - type: object
+          properties:
+            _id:
+              type: string
+              description: The application's unique internal ID.
+              example: "633209ffd3bd3478005d1bac"
+            client_app_id:
+              type: string
+              description: The application's public App ID.
+              example: "myapp-abcde"
+            domain_id:
+              type: string
+              description: The application's associated domain ID.
+              example: "63320a2b5f9de9a6e0a213e8"
+            group_id:
+              type: string
+              description: |-
+                The application's [Atlas Project/Group ID](https://docs.atlas.mongodb.com/tutorial/manage-projects/).
+              example: "5b2ec991973129243223a114"
+            last_used:
+              type: integer
+              description: The time this app was last used in [UNIX time](https://en.wikipedia.org/wiki/Unix_time) (i.e. the number of seconds since Jan 1, 1970).
+              example: 1664224746
+            last_modified:
+              type: integer
+              description: The time this app was last modified in [UNIX time](https://en.wikipedia.org/wiki/Unix_time) (i.e. the number of seconds since Jan 1, 1970).
+              example: 1656440824
+            product:
+              type: string
+              description: The product this app is for.
+              enum:
+                - standard
+                - atlas
+                - data-api
+    AppEnvironment:
+      type: string
+      description: The application's environment. An empty string indicates that the app does not have a specified environment.
+      example: "production"
+      enum:
+        - ""
+        - "development"
+        - "testing"
+        - "qa"
+        - "production"
+    DeploymentModel:
+      type: string
+      description: An application [deployment model](https://www.mongodb.com/docs/atlas/app-services/manage-apps/deploy/deployment-models-and-regions/#deployment-models).
+      enum:
+        - "GLOBAL"
+        - "LOCAL"
+    CloudProviderRegionId:
+      oneOf:
+        - $ref: "#/components/schemas/AwsDeploymentRegion"
+        - $ref: "#/components/schemas/AzureDeploymentRegion"
+        - $ref: "#/components/schemas/GcpDeploymentRegion"
+    AwsDeploymentRegion:
```

Comment:
Oh boy. I guess it makes sense to provide this here, but we also need to remember to update it when we add new regions. So far, I think we've just been adding them to the [Deployment Models & Regions page](https://www.mongodb.com/docs/atlas/app-services/manage-apps/deploy/deployment-models-and-regions/#cloud-deployment-regions).

Looking at that Deployment Regions page, should we somehow specify here which ones are restricted to local only? Or maybe just add something in the description that some are only available for local deployment, and to consult the docs?

It also looks like we try to provide values for this in the [App config reference](https://www.mongodb.com/docs/atlas/app-services/manage-apps/configure/config/app/), but those are also not up-to-date. Should we make a separate ticket to update this there, or do you want to tag that onto this PR?

Reply:
I think we should get into the habit of assuming that any configurable change can/will have corresponding Admin API docs updates.

> some are only available for local deployment, and to consult the docs

I think this makes the most sense

> looks like we try to provide values for this in the [App config reference](https://www.mongodb.com/docs/atlas/app-services/manage-apps/configure/config/app/), but those are also not up-to-date

good catch I'll tag on in this PR